rosetta/docs/NATIVE_FP8.md

```diff
@@ -128,7 +128,61 @@ python -m paxml.main \
 
 Please ensure you include the first two flags, `--xla_gpu_enable_reduction_epilogue_fusion=false` and `--xla_gpu_enable_triton_gemm=false`, as they are essential for enabling the FP8 functionality. The additional flags primarily focus on performance enhancement and should also prove beneficial for non-FP8 executions.
 
-### Transformer Engine vs Native FP8 Support
+## Guide for Ninja Users
+
+### Exact pattern that XLA can match for FP8 MatMul
+The specific graph pattern that XLA supports for FP8 matmul is illustrated below:
+
+```
+  convert -> multiply -> (x) -> dot
+broadcast ->
+
+# or
+
+  convert -> (x) -> dot
```

Comment:
Can you confirm we don't fuse anything else, like the bias addition?

Reply:
I think bias and other epilog ops work same with the dot of the higher precision (e.g. bf16, fp32). We do support epilog quantize but it is complex and deserve a special section.

Reply:
And on a second thought, I don't mention it here is the bias (or other epilog) won't stop us using the fp8. I mean XLA will do a good job on whether to fuse the epilog to dot, like if they have consumers/ spmd insertions. Users shouldn't worry too much on it. But for the input quantize, users need to be careful to follow the above pattern, otherwise they will not use fp8 gemm.

Reply:
It can be in another PR. But having an example that show the bias fusion to have at least one example with epilog fusion would be great.

Reply:
Done.